src/librustc_mir/borrow_check/conflict_errors.rs

```diff
@@ -105,7 +105,9 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                 format!("{} occurs due to use{}", desired_action.as_noun(), use_spans.describe()),
             );
 
-            err.buffer(&mut self.errors_buffer);
+            // Bypass the buffer for this error, so we don't downgrade it to a
```

Comment:
Comment style very-much-nit: It's more readable to use a semantic linefeed style where you make a newline after the comma.

https://sembr.org/